app/uk/gov/hmrc/agentsexternalstubsfrontend/views/edit_user.scala.html

```diff
@@ -397,7 +397,7 @@ <h2>Stride roles</h2>
             @if(userForm.data.get("affinityGroup").contains("Agent")) {
                 <a class="govuk-body govuk-link" name = "suspendedRegimes"></a>
                 <h2>Suspended Regimes</h2>
-                <span class="govuk-hint">Only Agents can be suspended. Valid Regimes: ITSA, VATC, TRS, CGT, PPT, IRV, ALL, AGSV</span>
+                <span class="govuk-hint">Only Agents can be suspended. Valid Regimes: ITSA, VATC, TRS, CGT, PPT, PIR, ALL, AGSV</span>
```

Comment:
I don't think we should have PIR or IRV here because these are regimes that DES will return and DES will never return either of those. IRV/PIR suspensions are enabled when the regime is either AGSV or ALL

Reply:
PR updated: IRV removed in this hint, as it will be implicitly included for ALL/AGSV.